### PR TITLE
Adding post url to cybersixgill sub alerts

### PR DESCRIPTION
**Pull request by shahul-loginsoft**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by adi88d** [comment]
@shahul-loginsoft
Please fill the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA) so we can start the review
thanks

**Comment by shahul-loginsoft** [comment]
I added release notes but still run validations are failing. can please check this?

**Comment by kgal-pan** [comment]
> I added release notes but still run validations are failing. can please check this?

@shahul-loginsoft - Please ignore, It's a known issue.

File: Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json [hunk 1]
```diff
@@ -1,31 +1,31 @@
 {
```
Comment (kgal-pan) [inline Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json:1]: Please revert this as content hasn't been changed.

File: Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.py [hunk 2]
```diff
@@ -117,6 +117,13 @@ def add_sub_alerts_fields(incident, item_info, sixgill_alerts_client):
         'cybersixgillactor': content_item.get('creator', None),
         'cybersixgilltriggeredassets': triggered_assets
     })
+    if item_info.get("sub_alerts_length"):
+        url = "https://portal.cybersixgill.com/#/alerts?"
+        content = f"actionable_alert_content_id={item_info.get('id')}&"
+        index = f"aggregatedIndex={item_info.get('aggregate_alert_id')}&"
+        aleid = f"filters.alert_id={item_info.get('id')}"
+        post_url = f"{url}{content}{index}{aleid}"
+        incident["CustomFields"].update({"cybersixgillposturl": post_url})
```
Comment (kgal-pan) [inline Packs/Cybersixgill-ActionableAlerts/Integrations/CybersixgillActionableAlerts/CybersixgillActionableAlerts.py:126]: Please remove this and add it to the mapper.

**Comment by kgal-pan** [comment]
New PR in https://github.com/demisto/content/pull/31632

**Pull request closed without merging**